src/main/java/com/koapti/lazychef/model/mappers/entity/UserEntityMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.koapti.lazychef.model.mappers.entity;
+
+import com.koapti.lazychef.model.dto.UserDto;
+import com.koapti.lazychef.model.entity.User;
+import com.koapti.lazychef.model.types.RoleType;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class UserEntityMapper {
+    public static User toUserEntity(final com.koapti.lazychef.api.model.User apiUser) {
+        User user = new User();
+        user.setLogin(apiUser.getLogin());
```

Comment:
moze lepiej jakiegos buildera niz settery?

Reply:
To jest entity, nie można dodawać builderów tak łatwo, jak się wydaje xd